element/daterange/styles.css

```diff
@@ -0,0 +1,3 @@
+.datestring {
+    padding-bottom: 1.5rem!important;
```

Comment:
1. In my test setup, this css rule wasn't delivered to the browser at all. I assume that styles.css files aren't processed in subplugins of this plugin.
2. After added to the page manually, there was no difference when the rule had !important set and when it had not. I assume that !important would be overkill here.
3. I understand why you added this rule, but I am afraid that it does not improve the form. I would rather propose to omit this rule for now.